src/site/content/en/blog/tools-for-extracting-critical-css/index.md

```diff
@@ -0,0 +1,86 @@
+authors:
+  - mihajlija
+date: 2019-05-07
+hero: hero.jpg
+alt: A description of the hero image for screen reader users.
+description: |
+  Learn how to improve render times with critical CSS technique and how to choose the best tool for your project.
+tags:
+  - post # post is a required tag for the article to show up in the blog.
+  - performance
+  - css
+---
+
+The browser must download and parse CSS files before it can show the page, which makes CSS a render-blocking resource. If CSS files are big, or network conditions are poor, requests for CSS files can significantly increase the time it takes for a web page to render.
+
+{% Aside 'key-term' %}
+Critical CSS is a technique that extracts the CSS for above-the-fold content in order to render content to the user as fast as possible.
+{% endAside %}
+
+<figure class="w-figure  w-figure--center">
+  <img class="" src="./above-the-fold.png" alt="An illustration of a laptop and a mobile device with web pages overflowing the edges of screens" style="max-width: 600px;">
+</figure>
+
+{% Aside 'note' %}
+Above-the-fold is all the content a viewer sees on page load, before scrolling. There is no universally defined pixel height of what is considered above the fold content since there is a myriad of devices and screen sizes.
+{% endAside %}
+
+Inlining extracted styles in the `<head>` of the HTML document eliminates the need to make an additional request to fetch these styles. The remainder of the CSS can be loaded asynchronously.
+
+<figure class="w-figure  w-figure--center">
+    <img class="w-screenshot" src="./inline-critical-css.png" alt="HTML file with critical CSS inlined in the head">
+    <figcaption class="w-figcaption">
+    Inlined critical CSS
+    </figcaption>
+</figure>
+
+Improving render times can make a huge difference in [perceived performance](https://developers.google.com/web/fundamentals/performance/rail#ux), especially under poor network conditions. On mobile networks, high latency is an issue regardless of bandwidth.
+
+<figure class="w-figure  w-figure--center">
+  <img class="w-screenshot" src="./filmstrip-comparison.png" alt="Filmstrip view comparison of loading a page with render-blocking CSS (top) and the same page with inlined critical CSS (bottom) on a 3G connection. Top filmstrip shows six blank frames before finally displaying content. Bottom filmstrip displays meaningful content in the first frame.">
+  <figcaption class="w-figcaption">
+    Comparison of loading a page with render-blocking CSS (top) and the same page with inlined critical CSS (bottom) on a 3G connection
+  </figcaption>
+</figure>
+
+If you have poor [First Contentful Paint](https://web.dev/first-contentful-paint/) (FCP) and see "Eliminate render-blocking resource" opportunity in Lighthouse audits it’s a good idea to give critical CSS a go.
+
+<img class="w-screenshot" src="./lighthouse-audit.png" alt="Lighthouse audit with 'Eliminate render-blocking resource' or 'Defer unused CSS' opportunities">
```

Comment:
I was wondering if you have a higher res version of this image? The text seems a bit fuzzy. We'll run the image through compression before it goes live so you don't have to worry much about compressing beforehand. So long as images are sized correctly, we should be able to handle things after that.